### ISSUE TITLE
Download buttons don't respond

### ISSUE THREAD

**Issue by donderden** [schneidermanuel/TwitchLeecher-Dx#25]
**Description**
①The VOD download button that came up in Channnel Name Search don't respond. Download will not start.
②The names of channels registered in Favorite Channels are invisible with white background and white letters.It becomes visible by mouse over.

**Information**
No log file
-Operating System
Windows7 SP1 64Bit
.NET Runtime 6.0.27 (x64)
.NET SDK 6.0.421 (x64)
TwitchLeecher-Dx ver3.5.6

**Additional information that might be useful**
I'm afraid this is a difficult problem. Please let me know if any programs are missing.
Thank you.
![SnapCrab_Twitch Leecher DX 356_2024-5-4_1-1-53_No-00](https://github.com/schneidermanuel/TwitchLeecher-Dx/assets/168849024/b3176bb6-3a4f-4579-8798-b8e5ec7edd2a)
![SnapCrab_Twitch Leecher DX 356_2024-5-4_1-0-49_No-00](https://github.com/schneidermanuel/TwitchLeecher-Dx/assets/168849024/6c1391c7-a70b-40b3-ab74-f0c7889faf15)

**Comment by schneidermanuel** [comment]
The second problem only occurs when you system is set to light mode. 
A fix is already implemented, but not published. 
The first problem might be something with Windows7, I never tested it on it.

**Comment by schneidermanuel** [comment]
Have you tried setting Download and Temporary Folder in the Settings to something different than what you got there? 
Is it wotking when both are set to a "simple" path like H:\Test ?

**Comment by donderden** [comment]
I've changed the settings for both folders to the root of the H drive, to a simple path (H:\), but the trouble is still the same.
Thanks for the reply.

**Comment by schneidermanuel** [comment]
Might you provide me with the channel name used in your search?

**Comment by donderden** [comment]
I get the same trouble no matter what channel I search.
Here are some of the channels I have tried.

https://www.twitch.tv/fps_shaka
https://www.twitch.tv/shibuya_kaho
https://www.twitch.tv/kokujintv

**Comment by schneidermanuel** [comment]
I am afraid I am unable to reproduce this. With the newest version, are you able to cklick the thumbnail and open the video? 
It might be Windows 7 related, I wasn't really able to test it there

**Comment by donderden** [comment]
Yes. In the latest version 3.6.0, I can click the thumbnail and open the video in browser.
But when I press the download button, it does not respond.
I have recorded the video ,please watch it.
Thanks for the reply.
https://youtube.com/shorts/em0w0q6Edss?feature=share
